frontend/src/utils/api.ts

```diff
@@ -9,5 +9,6 @@ export const api = {
     EXPORT: '/api/export/',
     CHANGE_PASSWORD: '/api/change_password/',
     VIEW_SCHEDULED_EVENTS: '/api/scheduled_event/',
+    CREATE_SCHEDULED_INSTANCE: '/run_instances/',
```

Comment:
I'm certain this isn't the path ... use the full path for this endpoint ... it's confusing to have it depend on another endpoint & complicates changes.

Reply:
Updated with a template string that takes in the `scheduleId`